src/types.rs

```diff
@@ -108,17 +145,26 @@ impl FunctionInfo {
     }
 }
 
-#[derive(Default)]
+#[pyclass]
+#[derive(Default, Clone)]
 pub struct Request {
+    #[pyo3(get, set)]
     pub queries: HashMap<String, String>,
+    #[pyo3(get, set)]
     pub headers: HashMap<String, String>,
     pub method: Method,
+    #[pyo3(get, set)]
     pub params: HashMap<String, String>,
-    pub body: Bytes,
+    #[pyo3(get, set)]
+    pub body: ActixBytesWrapper,
```

Comment:
Same for `headers` and `body` here - you can use the getter and setters.

Additionally, we can create a custom class over hashmap for `Headers` and override `__getattr__` and `__setattr__` methods in pyo3. 

Or extend the `dict` class?